qaframework-bdd-tests/src/test/resources/features/refapp-2.x/016-XssVulnerability/xssVulnerability.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Check Xss Vulnerability
+
+  Background:
+    Given user goes to active visit search page
+   
+  @selenium
+  @xssVulnerability
+  Scenario: Check XSS Vulnerability
+    When user searches for patient using patient identifier
+    And user goes to patient dashboard of last Active visit
```

Comment:
I think we can have these two steps under background. something like

```js
Background:
    Given user goes to active visit search page
    When user searches for patient using patient identifier
    Then the system goes to patient dashboard of last Active visit

Reply:
I don't see any reason why we are adding  it in background

Reply:
the scenario should show the actual steps involved in checking xss so those two steps suits to be within the background part.

Reply:
I really don't see any strong reason. What am seeing here is initiateLogin. All other scenario fits best in part of scenarios not in background. Let's try to use background part appropriately. Adding. Scenarios in background simply means that there are some scenarios you ensure to be run first.


On my scenario.  I expect initiateLogin to first run after every scenario. And this have been achieved with in before method. So it won't look a bit practical.

Reply:
```suggestion
    Then the system loads patient dashboard page of last Active visit
```

Reply:
@kdaud  does the language looks fine. i mean the scenario